src/domains/citizen-auth-common/03_storage.tf

```
@@ -277,3 +277,25 @@ resource "azurerm_storage_queue" "profiles_to_sanitize" {
   name                 = "profiles-to-sanitize"
   storage_account_name = module.io_citizen_auth_storage.name
 }
+
+
+module "azure_storage_account" {
+  source = "github.com/pagopa/dx//infra/modules/azure_storage_account?ref=main"
+
+  environment                          = local.itn_environment
+  resource_group_name                  = data.azurerm_resource_group.data_rg_itn.name
+  tier                                 = "l"
+  subnet_pep_id                        = data.azurerm_subnet.subnet_pep_itn.id
```

After initializing the module common_values, we can get the pep snet id from it

```suggestion
  subnet_pep_id                        = module.common_values.pep_subnets.itn.id
```